src/test/java/com/epam/tc/hw5/common/WebdriverSetUp.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.tc.hw5.common;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeClass;
+
+public class WebdriverSetUp {
+    public WebDriver driver;
+    public WebDriverWait wait;
+    public HomePage homePage;
+
+    @BeforeClass
```

Comment:
It's better to use any of Cucumber hooks instead of TestNG precondition annotation. It looks like this class must be removed, as below there is driver setup using cucumber hooks